apps/web/components/booking/BookingListItem.tsx

```diff
@@ -80,15 +168,42 @@ function BookingListItem(booking: BookingItem) {
     {
       id: "reschedule",
       label: t("reschedule"),
-      href: `/reschedule/${booking.uid}`,
       icon: ClockIcon,
+      actions: [
```

Comment:
New functionality of BookingListItem, now if they have actions prop they display now like a dropDown